# Camera intrinsics for D435 RGB from rs-enumerate-devices -c.
# Units in pixels. Distortion model 'plumb_bob' (Inverse Brown-Conrady distortion).
# If override_intrinsics_from_yaml:=true, cv_module.camera_info_override will republish these values
# with timestamps aligned to incoming images.

image_width: 1920
image_height: 1080
camera_name: d435
camera_matrix:
  rows: 3
  cols: 3
  data: [1369.25317382812, 0.0, 972.777648925781,
         0.0, 1368.90344238281, 559.209411621094,
         0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.0, 0.0, 0.0, 0.0, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0,
         0.0, 1.0, 0.0,
         0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [1369.25317382812, 0.0, 972.777648925781, 0.0,
         0.0, 1368.90344238281, 559.209411621094, 0.0,
         0.0, 0.0, 1.0, 0.0]
